.cirrus.yml

```diff
@@ -120,9 +111,10 @@ test_task:
 
     depends_on:
         - gating
-        - podman
 
-    script: "$SCRIPT_BASE/test.sh"
+    script:
+        - "$SCRIPT_BASE/latest_podman.sh"
+        - "$SCRIPT_BASE/test.sh"
```

Comment:
This will work, but as you can see all the output is jammed together.  You can make Cirrus-CI do multiple output "sections" (each with it's own drop-down) by:

```yaml
    install_latest_script: "$SCRIPT_BASE/latest_podman.sh"
    test_script: "$SCRIPT_BASE/test.sh"
```

This will result in one section called "install_latest" and another called "test"  :smile: